packages/ec-core/src/distributions/one_of_macro.rs

```diff
@@ -46,7 +46,7 @@ macro_rules! uniform_distribution_of {
      (<$output_type:ty>  $($items:expr),+ $(,)?) => {
           // FIXME: is unwrap_unchecked a performance gain here?
           {
-               #[allow(clippy::unwrap_used)]
+               // The macro pattern guarantees that there is at least one item ($(...)+) and into_distribution for all types only errors if no element is present.
```

Comment:
the lint is ignored in macros anyways since as a user there is no reasonable way to silence it from the outside - the expect expects a lint to be triggered there tho, so that wouldn't work - a comment is probably the easiest solution, since no actual lint needs to be silenced